client/src/components/PillTopic/Pill.test.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { shallow, configure } from "enzyme";
+import PillTopic from "./PillTopic.js.js";
```

Comment:
Here it's called `PillTopic`, but the file is called `Pill` and the component is called `Pill`. Can you make sure this works?